docs/src/content/docs/fr/guides/css-and-tailwind.mdx

```diff
@@ -269,6 +272,34 @@ Si définies, les propriétés personnalisées CSS suivantes remplaceront les st
 }
 ```
 
+### Utiliser plusieurs configurations Tailwind
+
+Plusieurs configurations Tailwind peuvent être utilisées pour appliquer différents styles à différentes parties de votre site, par exemple lorsque vous [utilisez Starlight dans un sous-chemin](/fr/manual-setup/#utiliser-starlight-dans-un-sous-chemin) ou lorsque vous ajoutez des [pages personnalisées](/fr/guides/pages/#pages-personnalisées) à votre site.
+Par exemple, vous souhaiterez peut-être utiliser les styles de réinitialisation Preflight de Tailwind dans vos pages personnalisées, tout en appliquant la couche de compatibilité de Starlight aux pages Starlight.
+
+La configuration Tailwind CSS suivante configure Tailwind sans aucun module d'extension ni configuration supplémentaire et peut être utilisée comme point de départ pour les pages non Starlight :
```

Comment:
```suggestion
La configuration Tailwind CSS suivante configure Tailwind sans aucun module d'extension ni configuration supplémentaire et peut être utilisée comme point de départ pour les pages non gérées par Starlight :
```

Only a suggestion, not sure it's better and no strong opinion about it, just mentioning it as the current version felt a bit weird to me when reading it. Up to you to discard or accept.